Backend/BikesRentalServer.Services/BikesService.cs

```diff
@@ -191,6 +201,8 @@ public ServiceActionResult<Reservation> ReserveBike(string id)
                 return ServiceActionResult.InvalidState<Reservation>("Bike is rented");
             if (bike.Station.Status is StationStatus.Blocked)
                 return ServiceActionResult.InvalidState<Reservation>("Station is blocked");
+            if(_reservationsRepository.GetActiveReservation(id) != null)
```

Comment:
Dla lepszego stylu - spacja między `if`em a nawiasem.
Dla spójności z tym jak robimy wszędzie indzie - `is not null`